NativeOp.cpp

```diff
@@ -752,7 +752,11 @@ void* _malloc(size_t num_bytes) {
     //auto* stream = context->op_device_context()->stream();
     Allocator* allocator =
         context->device()->GetAllocator(AllocatorAttributes());
-    void* ptr = (void*)allocator->Allocate<uint8_t>(num_bytes);
+#if (TF_MAJOR_VERSION == 1 && TF_MINOR_VERSION >= 14)
```

Comment:
Can you add maybe a comment? `Allocate` was removed from the API since TF 1.14?

Reply:
That's right. I'll add a comment.